invenio_records_resources/resources/files/config.py

```diff
@@ -27,3 +28,5 @@ class FileResourceConfig(ResourceConfig):
         "item-commit": "/files/<key>/commit",
         "list-archive": "/files-archive",
     }
+
+    request_extra_args = {"include_deleted": ma.fields.Boolean()}
```

Comment:
this is usually configured as schema, not dict, see https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/resources/args.py